MonoGame.Framework/Graphics/Effect/IEffectFog.cs

```diff
@@ -48,32 +10,31 @@ namespace Microsoft.Xna.Framework.Graphics
 	public interface IEffectFog
 	{
         /// <summary>
-        /// The floating point fog color.
+        /// Gets or set the fog color in floating point format.

```

Comment:
Lets not use "gets or sets".  The programmer knows what a property is and that it can get or set.  That is not useful information to them.